### PR TITLE
[MODULAR][READY] GAGS Hoodies

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
GAGSifies our hoodies, part two of the Lesser GAGSification.
image

Follows the standard set by part one, moving the berets; icons in the new master folder, code under customization. Minimal change functionally, aside from removing the preset colors from vendors (you cant set them to not be recolorable. Its weird.) and adding new Trimmed versions that can be customized as well.

## How This Contributes To The Skyrat Roleplay Experience
More customization, and less duplicated+recolored sprites

## Changelog

:cl:
qol: Hoodies now use GAGS greyscale system! You can customize them more yourself, rejoice!!!
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Todo: fix loadout paths
(REALLY? IT WAS ORIGINALLY JUST THE COLOR? NOT AT ALL DISTINGUISHING IT WAS A HOODIE, JUST F---ING SUIT/[COLOR]??????)

**Comment by ghost** [comment]
> 
> 
> Todo: fix loadout paths (REALLY? IT WAS ORIGINALLY JUST THE COLOR? NOT AT ALL DISTINGUISHING IT WAS A HOODIE, JUST F---ING SUIT/[COLOR]??????)

Thank you mobile alt account, this is now **Complete**

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm [hunk 1]
```diff
@@ -346,58 +346,68 @@ GLOBAL_LIST_INIT(loadout_exosuits, generate_loadout_items(/datum/loadout_item/su
 
 
 //HOODIES
+/datum/loadout_item/suit/hoodie/greyscale
+	name = "Greyscale Hoodie"
+	item_path = /obj/item/clothing/suit/toggle/jacket/hoodie
+	can_be_greyscale = TRUE
```
Comment (ghost) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm]:
```suggestion
```

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm [hunk 2]
```diff
@@ -346,58 +346,68 @@ GLOBAL_LIST_INIT(loadout_exosuits, generate_loadout_items(/datum/loadout_item/su
...
 
-/datum/loadout_item/suit/hoodie/grey
-	name = "Grey Hoodie"
-	item_path = /obj/item/clothing/suit/storage/toggle/hoodie
+/datum/loadout_item/suit/hoodie/greyscale_trim
+	name = "Greyscale Trimmed Hoodie"
+	item_path = /obj/item/clothing/suit/toggle/jacket/hoodie/trim
+	can_be_greyscale = TRUE
```
Comment (ghost) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm]:
```suggestion
```

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm [hunk 3]
```diff
@@ -346,58 +346,68 @@ GLOBAL_LIST_INIT(loadout_exosuits, generate_loadout_items(/datum/loadout_item/su
...
 
-/datum/loadout_item/suit/black
+/datum/loadout_item/suit/hoodie/greyscale_trim_alt
+	name = "Greyscale Trimmed(Alt) Hoodie"
+	item_path = /obj/item/clothing/suit/toggle/jacket/hoodie/trim/alt
+	can_be_greyscale = TRUE
```
Comment (ghost) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm]:
```suggestion
```

**Review by GoldenAlpharex — APPROVED** [review]
Looks good.

**Pull request merged**